src/main/java/org/xstefank/verification/RepositoryFormatVerification.java

```diff
@@ -0,0 +1,21 @@
+package org.xstefank.verification;
+
+import org.xstefank.model.yaml.FormatConfig;
+
+public class RepositoryFormatVerification implements Verification {
+
+
+    private final String REGEX = "^[A-z,0-9].*/[A-z,0-9,-].*$";
+    private final String MESSAGE = "Bad repository format in configuration file";
+
+    public boolean verify(FormatConfig formatConfig) {
```

Comment:
please add `@Override` - https://stackoverflow.com/questions/94361/when-do-you-use-javas-override-annotation-and-why